### ISSUE TITLE
Error compiling 4.2.0 Xcode 15

### ISSUE THREAD

**Issue by josmdddeveloper** [wonderpush/wonderpush-ios-sdk#17]
Hello guys, the last update from Wonderpush gives me this error when archiving in Xcode:

```
ld: Undefined symbols:
  _OBJC_CLASS_$_WonderPushObjCInterop, referenced from:
       in WonderPushObjC.o
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

I tried to clean build folder. It installs fine in the device, but doesn't let me archive it. How could i solve it? Thanks in advance.

**Comment by nenadvulic** [comment]
same here !

**Comment by MateuszW13** [comment]
Do you use CocoaPods for deps? If yes you can try to update cocoapods to version to 1.13.0 and it might resolve the issue 😄

**Comment by josmdddeveloper** [comment]
Thank you @MateuszW13 I use Swift Package Manager.

I tried to use the new Xcode 15.1 beta and get the same error. Installing to Simulator and Physical device works great, but archiving is giving me the error of the "Undefined symbols".

The only WonderPush version it lets me archive is 4.1.6 

I had installed CocoaPods in this project a long time ago, but migrated all to Swift Package Manager after few months. I add this information because could be relevant.
